packages/store-engine-fs/src/index.ts

```diff
@@ -82,14 +82,18 @@ export class FileEngine implements CRUDEngine {
     return fs.remove(this.storeName);
   }
 
-  public async create<T>(tableName: string, primaryKey: string, entity: any): Promise<string> {
+  public async create<EntityType = Object, PrimaryKey = string>(
+    tableName: string,
+    primaryKey: PrimaryKey,
+    entity: EntityType,
+  ): Promise<PrimaryKey> {
     if (entity) {
       const filePath = this.resolvePath(tableName, primaryKey);
       if (typeof entity === 'object') {
         try {
-          entity = JSON.stringify(entity);
+          entity = JSON.stringify(entity) as any;
```

Comment:
`JSON.stringify(entity)` is a string, so we might need to do `EntityType = Object | string` if possible?